crates/anstream/Cargo.toml

```diff
@@ -25,18 +25,19 @@ pre-release-replacements = [
 ]
 
 [features]
-default = ["auto", "wincon"]
+default = ["auto", "wincon", "utf8"]
 auto = ["dep:anstyle-query"]
 wincon = ["dep:anstyle-wincon"]
+utf8 = ["dep:utf8parse", "anstyle-parse/utf8"]
```

Comment:
In general, I worry that this is an implementation detail and could change and you could lose the benefit